pages/edge/tutorials/index.mdx

```diff
@@ -0,0 +1,206 @@
+import { Steps, Callout, FileTree } from "nextra-theme-docs";
```

Comment:
This should be a static site file inside of tutorials. Not the index of tutorials